Changes

```diff
@@ -81,6 +81,9 @@ _______________
   grouping all the linker flags at the end of the C compiler commandline
   (David Allsopp and Samuel Hym, review by Nicolás Ojeda Bär)
 
+- #12996: Only link gcc_eh when available.
+  (Romain Beauxis, review by David Allsop and Miod Vallat)
```

Comment:
A very minor nit:
```suggestion
  (Romain Beauxis, review by David Allsopp and Miod Vallat)
```
FWIW I would either say "link libgcc_eh.a" or "link with -lgcc_eh" but it's only a Changs entry!!